app/views/world_locations/index.html.erb

```diff
@@ -1,67 +0,0 @@
-<% page_title sanitize(t("world_location.content.heading").gsub!(/&nbsp;/, " ")) %>
-<% page_class "govuk-width-container world-locations-index index-list-page" %>
-
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <header class="js-filter-list">
-      <%= render "govuk_publishing_components/components/title", {
-        title: sanitize(t("world_location.content.heading"))
```

Comment:
Can these locales be removed from the locale files now?  It'll save a bit of work when we do the tidy-up cards.